### DISCUSSION TITLE
RFP: Seeking Laravel Developers for Project Monitoring Information System (PMIS)

### DISCUSSION THREAD

**Discussion by tenzin7** [laravel/framework#52995] [Ideas]
Hello Laravel Community,

We are inviting proposals for the development of a Project Monitoring Information System (PMIS) for the Central Tibetan Administration. The system will be built using the Laravel framework, following our organization's web application development standards.

Key features include:

- Project tracking and reporting
- Approval matrix for department-level approvals
- Comment and notification system
- Centralized database with role-based security features
- Integration with existing systems (e.g., budget management)

Interested Laravel developers or vendors are encouraged to submit proposals. A detailed Request for Proposals (RFP) is 
attached, which outlines the requirements, features, and submission guidelines. For further clarification, feel free to reach out.

The submission deadline is 5 PM, 1 Nov 2024.

Thank you, and we look forward to hearing from you!

[RFP-PMIS.pdf](https://github.com/user-attachments/files/17199356/RFP-PMIS.pdf)
[TCRC Web Application Development Standards.pdf](https://github.com/user-attachments/files/17199358/TCRC.Web.Application.Development.Standards.pdf)

**Comment by macropay-solutions** [comment]
Hello,

We wold like to propose for analysis our paid solutions/libs to SPEED UP the implementation of this/these projects.

https://github.com/macropay-solutions/laravel-lumen-crud-wizard
https://github.com/macropay-solutions/laravel-lumen-crud-wizard-decorator

**Comment by tenzin7** [comment]
Hello,

Thanks for reaching out. We will review the libraries you've mentioned and
evaluate if they align with our project requirements and development
standards.
Should we find them suitable, we will consider incorporating them into the
solution.

In the meantime, please feel free to submit a formal proposal in response
to our RFP, outlining how your solutions can enhance the development
process, as well as any related costs or licensing fees.

Thank you once again for your offer. We look forward to hearing more from
you.


*Best regards,Tenzn Lodoe*

**Comment by macropay-solutions** [comment]
@tenzin7 we will not be sending you an official proposal because of the international restrictions on that area. But you are free to use our free version of the library: https://packagist.org/packages/macropay-solutions/laravel-crud-wizard-free
